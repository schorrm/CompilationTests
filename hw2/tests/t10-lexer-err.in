int main () {
    int foo = &x; //there aren't any pointers here?"
    return 0;
}