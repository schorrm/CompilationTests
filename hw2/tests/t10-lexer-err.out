31: Type -> INT
5: RetType -> Type
7: Formals -> epsilon
31: Type -> INT
line 2: lexical error
